react-native-in-app-payments-quickstart/app/Utilities.tsx

```diff
@@ -62,9 +62,9 @@ export function printCurlCommand(nonce: any, appId: any, verificationToken = und
   }
 }
 
-export async function showAlert(title: any, message = "", onPress = () => { }) {
+export async function showAlert(title: string, message = "", onPress = () => { }) {
   Alert.alert(
-    title,
+    title.toString(),
```

Comment:
if title is a string type, why do we still need a `.toString()`?

Reply:
Removed.